src/Extensions/Cabal.hs

```diff
@@ -67,19 +71,30 @@ __Throws__:
 parseCabalFileExtensions :: FilePath -> IO (Map FilePath [OnOffExtension])
 parseCabalFileExtensions cabalPath = doesFileExist cabalPath >>= \hasCabalFile ->
     if hasCabalFile
-    then ByteString.readFile cabalPath >>= parseCabalExtensions
+    then ByteString.readFile cabalPath >>= parseCabalExtensions cabalPath
     else throwIO $ CabalFileNotFound cabalPath
 
-{- | Parse default extensions from a @.cabal@ file content.
+{- | Parse default extensions from a @.cabal@ file content. This
+function takes a path to a @.cabal@ file. The path is only used for error
+message. Pass empty string, if you don't have a path to @.cabal@ file.
```

Comment:
Great docs! I need to add the same sentence to a similar function in `Extensions.Parser` :+1: